api/python/quilt3/packages.py

```diff
@@ -769,7 +979,7 @@ def _set_commit_message(self, msg):
 
         self._meta.update({'message': msg})
 
-    def build(self, name=None, registry=None, message=None):
+    def build(self, name=None, registry=None, message=None, serialization_dir=None):
```

Comment:
Here's another example where complexity creeps in. When objects are passed in (e.g., set, users will be ready to specify a path for serialization, but not during build).